DOKS-automatic-node-repair/README.md

```diff
@@ -1 +1,374 @@
+    Notice: Adding cluster credentials to kubeconfig file found in "~/.kube/config"
+    Notice: Setting current-context to do-lon1-mobius-testing-cluster
+    ID                                      Name                       Region    Version        Auto Upgrade    Status     Node Pools
+    11bdd0f1-8bd0-42dc-a3af-7a83bc319295    mobius-testing-cluster     lon1      1.21.2-do.2    false           running    basicnp
+    ```
+
+6. Get worker nodes status:
+
+    ```bash
+    kubectl get nodes
+    ```
+
+    The output looks similar to the following:
+    ```
+    NAME            STATUS   ROLES    AGE     VERSION
+    basicnp-8k4ep   Ready    <none>   2m52s   v1.21.2
+    basicnp-8k4es   Ready    <none>   2m16s   v1.21.2
+    ```
+    ### Note:
+
+      If for some reason the `kubectl` context was not set properly we can always fix it via:
+      
+      ```bash
+      doctl k8s cluster kubeconfig save "$CLUSTER_NAME"
+      ```
+
+## Digital Mobius
+
+### Overview
+
+[Digital Mobius](https://github.com/Qovery/digital-mobius) is an open source project written in the `GO` language which can automatically recycle unhealthy nodes from a `DigitalOcean Kubernetes` cluster (DOKS).
+
+### How it Works
+
+The logic inside the application is watching for nodes that are in an unhealthy state at a regular interval specified by the user. A node is considered to be unhealthy if the [Node Condition](https://kubernetes.io/docs/concepts/architecture/nodes/#condition) is set to `Ready` and the status to `False` or `Unknown`. `Digital Mobius` provides also a user configurable flag that specifies how much time has to pass before an unhealthy node needs to be recycled. If all the conditions enumerated previously are met then the affected node(s) will be re-created. It does so via a call to the DigitalOcean's Kubernetes [Delete Kubernetes Node](https://docs.digitalocean.com/reference/api/api-reference/#operation/delete_kubernetes_node) `REST API`.
+
+Below is a diagram showing how `Digital Mobius` works:
+
+![Digital Mobius Flow](content/img/digital-mobius-flow.png)
+
+### Configuration Overview
+
+In order for the application to work it needs a set of environment variables to be configured and available as seen in the [Helm Chart](https://github.com/Qovery/digital-mobius/blob/main/charts/Digital-Mobius/values.yaml) values file (explanations inline):
+
+```
+LOG_LEVEL: "info"
+DELAY_NODE_CREATION: "10m"                                  # a worker node gets recycled after being unhealthy for this period of time
+DIGITAL_OCEAN_TOKEN: "<your_digital_ocean_api_token>"       # personal DO API token value
+DIGITAL_OCEAN_CLUSTER_ID: "<your_digital_ocean_cluster_id>" # DOKS cluster ID that needs to be monitored
+```
+
+The most important values that we need to here provide is the DigitalOcean `REST API token`, the `Cluster ID` and the `Node Creation Delay` time interval which can be expressed in seconds or minutes using the appropriate suffix - e.g.: `10s`, `10m`, etc. 
```

Comment:
nit: we tend to just call it "API token" since its scope isn't restricted to the REST API.

Reply:
Thanks for the suggestion. Fixed in the latest commit.